### DISCUSSION TITLE
Welcome to secrets-store-csi-driver-provider-azure Discussions!

### DISCUSSION THREAD

**Discussion by aramase** [Azure/secrets-store-csi-driver-provider-azure#343] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AkshayaAnil** [comment]
Hey Aramase,  I am exploring this project for a use case to  mount azure kv secrets to pods, I got a question though, when the secrets are  mounted as volume using the csi driver, is it going to live on the disk(of the node where pod is running) until pod is terminated ? This is an important question for my implementation. kindly let me your comment.
